third_party/forked/shell2junit/sh2ju.sh

```diff
@@ -101,37 +101,37 @@ function juLog() {
   # eval the command sending output to a file
   outf=/var/tmp/ju$$.txt
   errf=/var/tmp/ju$$-err.txt
-  >${outf}
+  :>${outf}
```

Comment:
[SC2188](https://github.com/koalaman/shellcheck/wiki/SC2188) - _"This redirection doesn't have a command. Move to its command (or use 'true' as no-op)."_

Essentially using true as a noop here. As far as I can gather, it's just used to create "${outf}"

Reply:
should we use touch then?

Reply:
I don't think it matters. It's still valid posix

Reply:
It's valid but I think less people will understand this as opposed to touch.